test/integration/thruster.cc

```diff
@@ -172,13 +196,9 @@ void ThrusterTest::TestWorld(const std::string &_world,
     if (_namespace == "custom")
       EXPECT_NEAR(0.0, pose.Rot().Roll(), 0.1);
     else
-      EXPECT_NEAR(0.0, pose.Rot().Roll(), _baseTol);
+      ASSERT_NEAR(0.0, pose.Rot().Roll(), _baseTol);
```

Comment:
Does this need to be an assertion? If the test fails here, it could be helpful for debugging to also see where else it fails